src/main/java/com/dreamteam/database/Main.java

```diff
@@ -331,30 +335,41 @@ public static void dailyAssetsReport(String date) {
 			pdf_writer.drawImage(asset_img, 20, 400);
 			pdf_writer.drawImage(sales_img, 20, 40);
 			pdf_writer.close();
+
 			PDPage stats_page = new PDPage(PDRectangle.A4);
 			document.addPage(stats_page);
 			pdf_writer = new PDPageContentStream(document, stats_page);
+
 			pdf_writer.beginText();
-			pdf_writer.moveTextPositionByAmount(100, 800);
+			pdf_writer.newLineAtOffset(100, 800);
+			pdf_writer.setLeading(23.0);
 			pdf_writer.setFont(font, 20);
 			pdf_writer.showText("Top Products ");
-			pdf_writer.moveTextPositionByAmount(0, -23);
+			pdf_writer.newLine();
+			pdf_writer.setLeading(17);
 			pdf_writer.setFont(font, 14);
+
 			LinkedHashMap<String, Double> top_products = ORDER_DATABASE.findTopProducts(date);
 			for (String id : top_products.keySet()) {
 				pdf_writer.showText(id + ", $" + top_products.get(id));
-				pdf_writer.moveTextPositionByAmount(0, -17);
+				pdf_writer.newLine();
```

Comment:
yes, and writer.setLeading(double) sets the pixel length to skip to the next line, otherwise everything prints on top of each other. This change is made to remove multiple deprecated warnings in our program.